src/index.ts

```diff
@@ -224,6 +224,28 @@ function b32decodeXemAddr(data: string): Buffer {
   return nemSdk.default.utils.convert.ua2hex(nemSdk.default.model.address.b32decode(address));
 }
 
+function ontHexToBase58(hexEncoded: Buffer): string {
+  let addr: Buffer;
+  const ADDR_VERSION = '17';
+  if(hexEncoded.toString('hex').length !== 40) {
+    throw Error('Unrecognised address format');
+  }
+  addr = Buffer.concat([new Buffer(ADDR_VERSION,'hex'), hexEncoded]);
```

Comment:
You can pass in the version as an int, and use an array of octets here - see for instance `makeBase58CheckEncoder`.